utils/get_data.py

```diff
@@ -17,11 +21,37 @@ def get_data(asset_type, payload):
         BASE_URL, asset_type,
         payload_encoded)
 
-    # # Get the JSON
+    # Get the JSON
     response = urllib2.urlopen(url)
     r = json.load(response)
     results = r["results"]
-    # count = r["count"]
 
     # Return the results
     return results
+
+
+def get_bbox(iface):
```

Comment:
I was in doubt whether to put it in the get_data module (getting the bbox as a form of getting data) or the geometry module.
The fact that you comment on this, shows that the geometry module is probably more logical.